jsonthebuilder/src/components/ResumeContainer.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import sampleResume from "../assets/sampleResume";
+import ResumeField from "./ResumeField";
+
+export default function ResumeContainer() {
+  // const sample = {sampleResume: "peos"};
+  // const [information, setInformation] = useState(JSON.parse(sample))
+  const jasson = JSON.stringify(sampleResume);
+  const obj = [JSON.parse(jasson)];
```

Comment:
@tammiec comments are on 🔥 

The parsing it's already done for you when you import the JSON, but if you want to wrap this into an array you can simple do:
```suggestion
const obj = [sampleResume];
```